explorer/app/components/Layout/components/Footer/footer.stories.tsx

```diff
@@ -67,19 +68,19 @@ AnvilFooter.args = {
     ],
     socials: [
       {
-        type: "discourse",
+        ...SOCIAL.DISCOURSE,
```

Comment:
Footer and Header stories' socials updated with `label` via the new `SOCIAL` const.